Tools/Scripts/webkitdirs.pm

```diff
@@ -38,6 +38,7 @@ use Digest::MD5 qw(md5_hex);
 use FindBin;
 use File::Basename;
 use File::Find;
+use File::Glob qw(bsd_glob);
```

Comment:
Hoping that this won't break any platforms by itself.

Reply:
`bsd_glob` isn't OS-specific, it's just behaves more like modern POSIX glob than perl's `glob` function does. The main reason I'm using it is to support SDK paths containing whitespace (though I suspect other parts of our build process do not). From perl's docs, emphasis mine:

> File::Glob::bsd_glob() implements the FreeBSD glob(3) routine, which is a superset of the POSIX glob() (described in IEEE Std 1003.2 "POSIX.2"). bsd_glob() takes a mandatory pattern argument, and an optional flags argument, and returns a list of filenames matching the pattern, with interpretation of the pattern modified by the flags variable.
>
> Since v5.6.0, Perl's CORE::glob() is implemented in terms of bsd_glob(). Note that they don't share the same prototype--CORE::glob() only accepts a single argument. **Due to historical reasons, CORE::glob() will also split its argument on whitespace, treating it as multiple patterns, whereas bsd_glob() considers them as one pattern**. But see :bsd_glob under ["EXPORTS"](https://perldoc.perl.org/File::Glob#EXPORTS), below.